lib/modules/commands/snek.rb

```diff
@@ -0,0 +1,15 @@
+module DrawBot
+	module DiscordCommands
+		module Actions
+			extend Discordrb::Commands::CommandContainer
+			command(:snek,
+				     description: 'Post a random cute snek!'
+				     usage: "#{BOT.prefix}snek") do |event|
+				response = Database::response.where(key: 'snek')
+
+				event << "#{response}"
+				event.message.delete
+			end
+		end
+	end
+end
```

Comment:
All of these commands in this PR need their indentation styling fixed.

Any indentation level should always be **two spaces**, with very few exceptions, like splitting a method call across the next line, in which you align the method (`.`) call operators.

Please use other reviewed parts of the bot for reference.